### PR TITLE
Fixed collision with mathsass log function

### PR DESCRIPTION
**Pull request by MasterOfZozo**
Regarding to issue #149

### CODE REVIEW

**Comment by eduardoboucas** [comment]
It should be `im-log` instead of `im_log`, as per the rest of the code base. Do you mind changing that?

Also, I'm thinking some of the plugins people developed could be using `log()` and will therefore break. How should we handle changes that are not breaking changes for the library but that might be for the plugins? Any thoughts, @HugoGiraudel?

**Comment by MasterOfZozo** [comment]
It seems that there is not a easy way how to solve it as namespaces are not supported by sass. In my opinion is needed to rewrite all known plugins not to break compatibility.

**Comment by eduardoboucas** [comment]
I understand that, and we've been through lengthy discussions about namespacing (see #32).

I suppose my question here is a bit wider than the scope of this particular PR, which is how best to propagate this type of change to plugins that are developed by the community and therefore out of our control. On the one hand, we don't want to be locked in because of those plugins, but on the other hand it's not fair to ask them to rewrite things overnight.

**Comment by HugoGiraudel** [comment]
I guess: check each published plugin individually to see if they used `log`.

**Comment by eduardoboucas** [comment]
I've checked the plugins that are listed on the README and they don't seem to use `log()`.

Once you're able to change the name of the function to use an hyphen instead of underscore, I'm happy to merge.

**Comment by MasterOfZozo** [comment]
@eduardoboucas Underscores changed to hyphens.

**Comment by eduardoboucas** [comment]
Great, thanks!

**Comment by eduardoboucas** [comment]
@HugoGiraudel happy for me to merge?

**Comment by HugoGiraudel** [comment]
Sure.

File: src/helpers/_no-media.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-///
+-///
```
Comment (eduardoboucas) [inline src/helpers/_no-media.scss]: This was a pain...

**Pull request merged**